### PR TITLE
edited get_all_history to past 10 records

### PR DESCRIPTION
**Pull request by vinatsx**

### CODE REVIEW

**Comment by vinatsx** [comment]
[Backend - Set up History API](https://trello.com/c/GxIu8lJ5/9-backend-set-up-history-api)

**Comment by vinatsx** [comment]
[Backend - Set up History API](https://trello.com/c/GxIu8lJ5/9-backend-set-up-history-api)

**Comment by vinatsx** [comment]
[Backend - Set up getstatus API](https://trello.com/c/GDEELrDy/10-backend-set-up-getstatus-api)

File: backend/GymFeatures/GymAPI.py [hunk 1]
```diff
@@ -11,35 +12,72 @@
 sys.path.append("../")
 
 gym_api = Blueprint("gym", __name__)
-DEFAULT_KEY_LOC = "5-409"
+DEFAULT_KEY_LOC = "5-122"
+DEFAULT_TELEGRAM_HANDLE = "smchead"
```
Comment (cjunxiang) [inline backend/GymFeatures/GymAPI.py:16]: consider pushing these CONSTS up to a higher level, into one common file.

**Pull request merged**